### PR TITLE
add build rule for android and ios players

### LINKED ISSUE #6441: Ensure the NuGet package comes with iOS & Android DLLs
As part of the 2.2 release, a few small changes need to be made to include the Android and iOS binaries in the MRTK NuGet package:
- [x] Update the scripts\packaging\createnugetpackage.ps1 script to build iOS and Android
- [x] Update the build CI to generate the iOS and Android dlls

### PR DESCRIPTION
**Pull request by davidkline-ms**
With the addition of the experimental Unity AR camera settings provider, MRTK has a need for precompiled Android and iOS players, when distributing via NuGet.

This change enables the build rules for these players. Once approved and merged, the nuspec files will be updated to include these player builds.

This is part 1 of the change that will resolve #6441

### CODE REVIEW

File: scripts/packaging/createnugetpackages.ps1 [hunk 1]
```diff
@@ -75,13 +75,34 @@ try
             Write-Error "Building InEditor WindowsStandalone32 Failed! See log file for more information $(Get-Location)\Logs\Build.InEditor.WindowsStandalone32.$($Version).log";
         exit($lastexitcode)
     }
+    Write-Output "============ Building InEditor WSA ============ "
```
Comment (macborow) [inline scripts/packaging/createnugetpackages.ps1]: The order in which the targets are built is different than in the CI template... which might be fine but I thought it's worth to point that out.

File: pipelines/templates/compilemsbuild.yml [hunk 2]
```diff
@@ -34,6 +34,42 @@ steps:
...
+    restoreNugetPackages: false
+    msbuildVersion: '15.0'
+    msbuildArguments: '/p:BuildProjectReferences=false'
+  displayName: 'Build InEditor WSA'
+
+- task: MSBuild@1
+  inputs:
+    solution: '$(Build.SourcesDirectory)\MSBuild\Projects\MixedRealityToolkit.sln'
+    msbuildArchitecture: 'x64'
+    platform: 'Android'
```
Comment (keveleigh) [inline pipelines/templates/compilemsbuild.yml]: Do we know if the build machines have Android and iOS support installed?

Comment (MenelvagorMilsom) [reply pipelines/templates/compilemsbuild.yml]: Good point. They do appear to have the Android NDK, but I can't see anything about iOS in any of the agent capabilities.

File: scripts/packaging/createnugetpackages.ps1 (hunk 1 above)
Comment (davidkline-ms) [reply scripts/packaging/createnugetpackages.ps1]: @andreiborodin suggested this ordering. I will work with him to ensure the CI template is appropriately updated.

**Pull request merged**